mongo/options/mongooptions.go

```diff
@@ -152,13 +163,22 @@ func (af *ArrayFilters) ToArrayDocument() (bson.Raw, error) {
 	}
 
 	idx, arr := bsoncore.AppendArrayStart(nil)
+	buf := new(bytes.Buffer)
+	enc := new(bson.Encoder)
```

Comment:
We should initialize these values in the test case funcs.

Reply:
Will skip this as it is not in a test.

Reply:
Ah, good catch. Please disregard.